package.json

```diff
@@ -28,6 +28,9 @@
     "deploy:docs": "gh-pages -d docs/build -m 'deploy docs [ci skip]'",
     "lint": "cross-env NODE_ENV=production eslint .",
     "lint:fix": "yarn lint --fix",
+    "prettier": "prettier --list-different \"**/*.{js,jsx,ts,tsx}\"",
+    "prettier:fix": "prettier --write \"**/*.{js,jsx,ts,tsx}\"",
```

Comment:
Did you test it under Windows? In another project I had trouble with glob patterns as they tried to match files from `C:\` and not from the current directory. The fix was to prefix the glob pattern with `./`.

Reply:
I did not.  Do you have a windows box for testing?  Perhaps @layershifter?

Note that I've escaped double quotes here, which means the globs will not be expanded by your shell (mac vs win) but rather by the node module that prettier uses under the hood, node-glob.  That module _should_ be platform agnostic, but I'm not 100% sure this is all correct.

Let me know how a Win test goes:

- edit a JS file, leaving a format mistake
- commit it (don't push)
- expect only that JS file to go through eslint and prettier
- repeat with a `ts` file, expecting tslint and prettier

Reply:
Works on a Windows machine, too.

My trouble with the glob pattern under windows seems to be related to the quoting of the glob pattern. Using double quotes seems to fix it